doc/source/tutorial/stats/continuous_studentized_range.rst

```diff
@@ -0,0 +1,27 @@
+.. _continuous-studentized_range:
+
+Studentized Range Distribution
+==============================
+This distribution has two shape parameters :math:`\nu>0.` and :math:`k>0` and support is :math:`q \geq 0.`
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*} f(q; k, \nu) = \frac{k(k-1)\nu^{\nu/2}}{\Gamma(\nu/2)2^{\nu/2-1}}\int_{0}^{\infty}\int_{-\infty}^{\infty}s^{\nu-1}e^{\nu s^2/2}s\phi(z)\phi(sq + z)[\Phi(sq + z) - \Phi(z)]^{k-2}\mathrm{d}z\mathrm{d}s \end{eqnarray*}
```

Comment:
```suggestion
    \begin{eqnarray*} f(q; k, \nu) = \frac{k(k-1)\nu^{\nu/2}}{\Gamma(\nu/2)2^{\nu/2-1}}\int_{0}^{\infty}\int_{-\infty}^{\infty}s^{\nu-1}e^{\nu s^2/2}s\phi(z)\phi(sq + z)[\Phi(sq + z) - \Phi(z)]^{k-2} \,dz \,ds \end{eqnarray*}
```
I don't think this needs mathrm, and the \, adds a bit of space.

Reply:
While the original code for that was indeed taken from Wikipeida, I made sure to go through it and ensure parity with what we actually have which is sourced from equations 3, 4 and 5 in the paper [here](https://www.scielo.br/pdf/cagro/v41n4/1981-1829-cagro-41-04-00378.pdf). The difference you noted (`s^{\nu-1}` in ours vs. `s^{\nu}`) seems to be an error on Wikipedia's part, because the source paper lists that term as we have it:
![paperterm](https://user-images.githubusercontent.com/45153623/114340591-c27fcb80-9b0c-11eb-9d33-5e049e6d96b6.PNG)
 (`x` is equivalent to our `s`)

Reply:
This is a confirmed error on Wikipedia - that version doesn't correctly integrate to the CDF.

| Our PDF | Wikipedia PDF |
|--|--|
|![Our PDF](https://user-images.githubusercontent.com/45153623/114474904-0d045500-9bac-11eb-913e-509c7cd705ee.png)| ![Wiki PDF](https://user-images.githubusercontent.com/45153623/114474985-37561280-9bac-11eb-9987-e972cecd765f.png) |

Reply:
On closer examination, Wikipedia wasn't incorrect. When they combined the above `f(x; v)` term with the rest of the PDF, they combined the x^v-1 with an existing x. See image:
![rev](https://user-images.githubusercontent.com/45153623/114802821-31e6fc80-9d53-11eb-9cfd-44ae52e0aeb2.png)

I think this simplification makes sense, I'm going to do it for our formulas as well.